pygfx/__init__.py

```diff
@@ -18,7 +18,13 @@
 
 from .utils.color import Color
 from .utils.load import load_mesh, load_meshes, load_scene
-from .utils.load_gltf import load_gltf, load_gltf_mesh, print_tree
+from .utils.load_gltf import (
+    load_gltf,
+    load_gltf_async,
+    load_gltf_mesh,
+    load_gltf_mesh_async,
+    print_tree,
```

Comment:
`print_tree` sounds like a developer tool? Is it necessary to import into Pygfx' main namespace? If so, lets come up with a more precise name.

Reply:
Yes, this is a developer tool. After loading a glTF scene, users might sometimes want to inspect the scene graph to locate specific nodes or objects, or to further manipulate the scene content.

This is especially useful when we have not yet fully implemented out-of-the-box support for glTF models. (For instance, users might need to find SkinnedMesh objects in the scene and update their skeletons.)

Reply:
What do you think about the name `print_scene_graph()`

Reply:
sounds good to me!